admin_pages/registrations/EE_Registrations_List_Table.class.php

```diff
@@ -881,15 +880,14 @@ public function column_TXN_paid(EE_Registration $item)
                     'espresso_transactions_view_transaction',
                     $item->transaction_ID()
                 )
-                    ? '<span class="reg-pad-rght"><a class="status-'
-                      . $transaction->status_ID()
-                      . '" href="'
-                      . $view_txn_lnk_url
-                      . '"  title="'
-                      . esc_attr__('View Transaction', 'event_espresso')
-                      . '">'
-                      . $item->transaction()->pretty_paid()
-                      . '</a><span>'
+                    ? '<span class="reg-pad-rght">
+                        <a class="status-' . $transaction->status_ID() . '" 
```

Comment:
If we have a class like "status status-' . $transaction->status_ID() . '" it would be great.

Because in the cases that we want to give an style to this a tag we can use the general status class.

Reply:
100% agree.
I'll look to see if anything like that exists and adjust accordingly.

Reply:
this is changed in `dev` branch, so I'm going to leave as is, else it will just cause an unnecessary conflict